RecoLocalMuon/GEMSegment/plugins/ME0SegmentAlgorithm.cc

```diff
@@ -319,15 +318,17 @@ void ME0SegmentAlgorithm::buildSegments(const ME0Ensemble& ensemble, const Ensem
     
     muonRecHits.push_back(newRH);    
   }
+  if (rechits.size() < minHitsPerSegment) return;
```

Comment:
this could have stayed at the entry to the function: rechits is const and the output (me0segs ) are not modified until later.